app/database/models/server/group.ts

```diff
@@ -80,4 +82,8 @@ export default class GroupModel extends Model implements GroupInterface {
     @lazy members = this.collections.
         get<UserModel>(USER).
         query(Q.on(GROUP_MEMBERSHIP, 'group_id', this.id));
+
+    @lazy isCurrentUserAMember = this.members.observe().pipe(switchMap((members) => {
+        return of$(Boolean(members.find((m) => m.id === SYSTEM_IDENTIFIERS.CURRENT_USER_ID)));
```

Comment:
This won't ever return true, the value of the constant is the string "currentUserId"

Also this lazy query (observable) how do you plan to use it in the _at mention component? Or will the at_mention have a children component that can make use of this?

Reply:
Whoops wasn't meant to have this in there yet - will do the mention highlighting separately as we're not saving groupMemberships yet.